.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "cSpell.words": ["trpc"]
```

Comment:
This seems unnecessary.

Reply:
My B -- I got tired of seeing the Blue Squiggly
Must have hit Workspace instead of User

Reply:
Removed here: [32e728e](https://github.com/echobind/bisonapp/pull/283/commits/32e728eb0766be7d4506cdea12f23ab1a86fc89a)